Jenkinsfile

```diff
@@ -350,20 +350,6 @@ pipeline {
         GOPATH = "${env.WORKSPACE}"
         SNAPSHOT="true"
       }
-      when {
```

Comment:
This stage only run on master branch and tags

Reply:
forget, I've seen it is only for testing

Reply:
but it is easy to add an expression to the when condition that it is always false, something like

```
// Disabled temporarily for testing
expresion { return false }
```